### DISCUSSION TITLE
DynamoDB Store Support

### DISCUSSION THREAD

**Discussion by maxswjeon** [openfga/community#472] [Ideas]
### Checklist

- [x] I agree to the terms within the [OpenFGA Code of Conduct](https://github.com/openfga/.github/blob/main/CODE_OF_CONDUCT.md).

### Describe the problem you'd like to have solved

In the [blog article from OpenFGA](https://openfga.dev/blog/fine-grained-news-2023-12) we could find that GoDaddy has made a succesful development and operation with DynamoDB Backend. As you know, relational databases are costly. It would be really good if we could deploy OpenFGA with DynamoDB Backend.

### Describe the ideal solution

It would be really great if the GoDaddy's implementation could be relased to the public, or something else.

**Comment by ArrichM** [comment]
There is also a related [blogpost from AWS](https://aws.amazon.com/de/blogs/database/performant-fine-grained-authorization-at-scale-powered-by-amazon-dynamodb/). If this would be considered for merging, I might be able to contribute this.
